anago

```diff
@@ -650,6 +646,9 @@ get_build_candidate () {
     # If the branch is a 3-part branch (ie. release-1.2.3)
     if [[ $RELEASE_BRANCH =~ $BRANCH_REGEX ]] && \
        [[ -n ${BASH_REMATCH[4]} ]]; then
+      ((FLAGS_official)) \
```

Comment:
I'm confused about how to use this for 3-part branches. Does this mean, when creating the branch, I have to supply `--official`? That seemed weird to me because it ends up creating a `-beta.1`, and I think when creating 2-part branches we don't supply `--official`.

Or is what I described the old behavior and this PR changes that?